### PR TITLE
added movie ticket booking website in html & css

### PR DESCRIPTION
**Pull request by Jaishree18**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ We'll organise into different streams here. One kind request for motivation: Ple
 * Comic Reading Site
 * Product Landing Page
 * Fashion E-Commerce Website
+* Movie Ticket Booking System
```
Comment (praveenscience) [inline README.md:22]: Won't this come in Full Stack?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Can you check this please? It should be under full stack.

**Comment by Jaishree18** [comment]
I have tried changing, can you please review again

**Review by praveenscience — APPROVED** [review]
👍

**Comment by praveenscience** [comment]
Thanks for your contribution. Optional Request: [Subscribe to my **YouTube Channel**](https://www.youtube.com/praveenscience?sub_confirmation=1), if you haven't already subscribed and share it with your friends too.

**Pull request merged**